comment/utils.py

```diff
@@ -31,7 +31,7 @@ def is_gravatar_enabled():
 
 def get_gravatar_img(email):
     if not is_gravatar_enabled() or not email:
-        return '/static/img/default.png'
+        return getattr(settings, 'COMMENT_DEFAULT_PROFILE_PIC_LOC')
```

Comment:
You may directly use `settings.COMMENT_DEFAULT_PROFILE_PIC_LOC` in place of using `getattr` everywhere.